Bin/Debug/settings-template.json

```diff
@@ -18,7 +18,8 @@
             "DisplayNamePrefix":"[AutomatedBot] ",
             "TradePollingInterval":800,
             "LogLevel":"Success",
-            "AutoStart": "true"
+            "AutoStart": "true",
+            "SchemaLang":"2 letter code for language schema should be translated in"
```

Comment:
You should set the default here to "en" as well.  You can add a comment using `//`, though I don't think even that is necessary.  Most people will have seen the two-letter ISO abbreviation for their own language before.